src/pages/Login.tsx

```diff
@@ -0,0 +1,106 @@
+import {ArrowBackIcon, EmailIcon, Icon} from '@chakra-ui/icons'
+import {
+    Button,
+    Center,
+    FormControl,
+    FormLabel,
+    Heading, HStack,
+    Input,
+    InputGroup,
+    InputRightAddon,
+    Text,
+    useToast,
+    VStack
+} from '@chakra-ui/react'
+import { Field, FieldProps, Form, Formik, FormikProps, FormikValues } from 'formik'
+import React from 'react'
+import { useNavigate } from 'react-router-dom'
+import { useFetch } from 'use-http'
+
+export default function Login() {
+    const {post, response} = useFetch()
+    const navigate = useNavigate()
+    const toast = useToast()
+    const [show, setShow] = React.useState(false)
+    const handleClick = () => setShow(!show)
+
+    const tryLogin = async (values: FormikValues) => {
+        const adminData = await post('/login', values)
+
+        if (!response.ok) {
+            toast({ title: adminData.message, description: response.status, status: response.ok ? 'success' : 'error' })
+            return
+        }
+
+        toast({ title: 'Logged in', description: response.status, status: response.ok ? 'success' : 'error' })
```

Comment:
We can remove the return on line 32 and nest lines 34+35 inside an else clause (also simplifying to status: 'success')